app/org/maproulette/framework/mixins/TaskParserMixin.scala

```diff
@@ -149,6 +149,7 @@ trait TaskParserMixin {
       // challenges and projects fields
       get[Option[String]]("challenge_name") ~
       get[Option[String]]("project_name") ~
+      get[Option[Int]]("project_id") ~
```

Comment:
There are foreign key constraints on tasks to challenge to project, so the project id will exist and `Option` is not needed here

Reply:
reviewHistoryParser creates a TaskReview, and its sql does not include the project id. This should remain an Option.